create_data_image.sh

```diff
@@ -1,25 +1,14 @@
 #!/bin/bash
 
-# Currently everything runs as root, but that is going to change.
-USER_UID=0
-USER_GID=0
-
-source ./su_command.sh
-
 TOTAL_SIZE=0
 
 echo "Gathering applications..."
-APPS=
-for app in apps/*
-do
-	if test -f $app
-	then
-		APPS="${APPS} $app"
-		SIZE=$(stat -Lc %s ${app})
+for app in apps/*; do
```

Comment:
Please don't change the code formatting (`for`...`do` on one or two lines) in patches; it makes it harder to review.

An advantage of the two-line style is that it is immediately obvious how it will be parsed: when I read `apps/*;`, there is already this nagging doubt in my mind whether the shell might treat the semicolon as part of the file name.

Reply:
I've rewritten this now.

In general, regarding the semicolon in `if` and `for`: it is idiomatic shell code.